portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -53,4 +59,29 @@ private String convertToJsonUsingGson(List<String> quotesToGson) {
     String json = gson.toJson(quotesToGson);
     return json;
   }
+
+public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String text = getParameter(request, "text-input", "");
```

Comment:
A good Java practice here is when you pass value parameters into a method, it's good to comment on which parameter is needed here. For example, the getParameter call here can be rewritten as:
String text = getParameter(/* httpServletRequest= */ request, 
                                           /* name= */ "text-input", 
                                           /* defaultValue= */ "")
This is so that reviewers would have a better idea of what the method takes in/what it does, without referring to the method implementation.
This is just something to keep in mind for the future though, no need to fix it here